spec/integration/ClientSpec.php

```diff
@@ -77,6 +77,56 @@ function it_receives_the_expected_response_when_sending_an_email_notification(){
 
     }
 
+    function it_receives_the_expected_response_when_sending_an_email_notification_with_vaild_emailReplyToId(){
+    
+        $response = $this->sendEmail( getenv('FUNCTIONAL_TEST_EMAIL'), 
+        getenv('EMAIL_TEMPLATE_ID'), 
+        [ "name" => "Foo" ],
+        '',
+        getenv('EMAIL_REPLY_TO_ID'));
+
+        $response->shouldBeArray();
+        $response->shouldHaveKey( 'id' );
+        $response['id']->shouldBeString();
+
+        $response->shouldHaveKey( 'reference' );
+
+        $response->shouldHaveKey( 'content' );
+        $response['content']->shouldBeArray();
+        $response['content']->shouldHaveKey( 'from_email' );
+        $response['content']['from_email']->shouldBeString();
```

Comment:
Any way you could test the reply-to string here?

Reply:
No its not in the response. If its a 200 it found it and its linked otherwise its a 400. This is really just checking we haven't altered the resposnse in anyway.